pkg/gce-pd-csi-driver/utils_windows.go

```diff
@@ -93,6 +93,11 @@ func getDevicePath(ns *GCENodeServer, volumeID, partition string) (string, error
 	return proxy.GetDiskNumber(deviceName, partition, volumeKey.Name)
 }
 
+func disableDevice(devicePath string) error {
+	// This is a no-op on windows.
+	return nil
```

Comment:
Technically, this can be supported, see https://social.technet.microsoft.com/Forums/en-US/d99024c1-75d0-4344-9cce-8a7b895d1885/how-to-safely-remove-hardware-under-hyperv-2008-r2?forum=winservercore But I was not able to repro instability under Windows.

Reply:
Ok, thanks. I'll get this in on linux first